README.Rmd

```diff
@@ -63,20 +107,17 @@ The tool development is an open source development process.
 
 The participants of Work Package 2.2 of the [United4Surveillance project][United4Surveillance] will be the maintainers of the repository and can contribute directly, third parties can contribute via pull requests.
 
-The maintainers will meet regularly and discuss pull request, issues and add new issues according to the collected requirements.
+The maintainers are meeting regularly to discuss pull requests, issues and to add new issues according to the collected requirements.
 
-Maintainers and contributers can work indepedently on issues.
+Maintainers and contributors can work independently on issues.
 
 For more information on how to contribute, please see our [contributing guidelines](https://github.com/United4Surveillance/signal-detection-tool/blob/main/.github/CONTRIBUTING.md).
 
 ## Roadmap
-The first version 0.1.0 of the tool was provided to european public health institutes for piloting in April 2024. The piloting phase ends in November 2024.  
-Our goal is to get as much feedback from the piloting countries as possible.
-Therefore, we adopt an agile development approach with the aim to ship improved versions as often and as early as possible.
-Some of the current features we want to develop are:  
+The initial version of the tool (0.1.0) was deployed in April 2024 to eleven European public health institutes for piloting, which is scheduled to conclude in November 2024. Our goal during the pilot phase is to gather extensive feedback from the participating countries. Therefore, we are adopting an agile development approach aiming to ship improved versions as often and as early as possible. By mid-November 2024, the latest version (0.5.0) was provided to the piloting institutes.
```

Comment:
The initial version of the tool (0.1.0) was deployed in April 2024 to eleven European public health institutes for piloting, which is scheduled to conclude in November 2024. -> The initial version of the tool (0.1.0) was deployed in April 2024 to eleven European public health institutes for piloting. The official piloting phase ended in November 2024. The tool can and is used after November 2024 and will still be developed further.
This would already be up to date as November 2024 will soon be in the past.

Our goal during the pilot phase is to gather extensive feedback from the participating countries. Therefore, we are adopting an agile development approach aiming to ship improved versions as often and as early as possible. By mid-November 2024, the latest version (0.5.0) was provided to the piloting institutes. ->
Just delete as not so important in a README